templates/core/index.html

```diff
@@ -15,198 +17,241 @@ <h1>Secure embedded devices</h1>
         <a class="p-button--positive" href="/engage/ubuntu-core-20-webinar">Watch the webinar</a>
       </p>
       <p class="u-sv3">
-        <a class="p-link--external p-link--inverted" href="https://assets.ubuntu.com/v1/b2c770ea-Ubuntu+Core20+Datasheet.pdf">Read the Ubuntu Core datasheet</a>
+        <a class="p-link--external p-link--inverted"
```

Comment:
the asset server isn't considered 'external' you can remove the `p-link--external`

Reply:
```suggestion
        <a class="p-link--inverted"
```